src/Microsoft.Diagnostics.Monitoring.RestServer/Operation/EgressOperationQueue.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Channels;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Monitoring.RestServer
+{
+    internal interface IEgressOperationQueue
```

Comment:
Is this interface needed? It looks like this is an implementation detail of the server assembly.